src/components/ErrorBoundary.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+class ErrorBoundary extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { hasError: false };
+    }
+
+    static getDerivedStateFromError() {
+        return { hasError: true };
+    }
+
+    render() {
+        const { hasError } = this.state;
+        const { children } = this.props;
+        if (hasError) {
+            return "Something went wrong";
+        }
```

Comment:
I think this currently wouldn't show up on the Home page because of the black text on black background lol

Reply:
www i'll figure that out later